### PR TITLE
chore(agw): add E1 to autopep8 python formatting

### PR DESCRIPTION
**Pull request by themarwhal**
Signed-off-by: Marie Bremner <marwhal@fb.com>

## Summary
See https://github.com/magma/magma/issues/11816 for details

Trying to add E1 (indentation related formatting to our CI/formatting script). Formatting in advance before pulling the trigger to avoid confusion.

This change is entirely automatically generated by autopep8
`autopep8  --select W191,W291,W292,W293,W391,E131,E2,E3,E1 -r --in-place ./`

## Test Plan
CI

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/deploy/roles/magma/files/nx_actions.py [hunk 1]
```diff
@@ -215,8 +215,8 @@ def parse(cls, buf):
         def serialize(self, buf, offset):
             data = self.serialize_body()
             payload_offset = (
-                    ofp.OFP_ACTION_EXPERIMENTER_HEADER_SIZE +
-                    struct.calcsize(NXAction._fmt_str)
+                ofp.OFP_ACTION_EXPERIMENTER_HEADER_SIZE +
```
Comment (ardzoht) [inline lte/gateway/deploy/roles/magma/files/nx_actions.py:218]: should we disable `W504`?

**Review by ardzoht — APPROVED** [review]
lgtm

File: lte/gateway/deploy/roles/magma/files/nx_actions.py (hunk 1 above)
Comment (themarwhal) [reply lte/gateway/deploy/roles/magma/files/nx_actions.py:218]: good call! will do this as a followup

Comment (themarwhal) [reply lte/gateway/deploy/roles/magma/files/nx_actions.py:218]: https://github.com/magma/magma/pull/11880

**Pull request merged**